crates/bevy_ecs/src/entity_disabling.rs

```diff
@@ -24,54 +70,132 @@
 use crate::{
     component::{ComponentId, Components, StorageType},
     query::FilteredAccess,
+    world::{FromWorld, World},
 };
 use bevy_ecs_macros::{Component, Resource};
+use smallvec::SmallVec;
 
 #[cfg(feature = "bevy_reflect")]
 use {crate::reflect::ReflectComponent, bevy_reflect::Reflect};
 
-/// A marker component for disabled entities. See [the module docs] for more info.
+/// A marker component for disabled entities.
+///
+/// Semantically, this component is used to mark entities that are temporarily disabled (typically for gameplay reasons),
+/// but will likely be re-enabled at some point.
+///
+/// Like all disabling components, this only disables the entity itself,
```

Comment:
For all lot of these comments here, I think you can just reference what you wrote at the top of the module. 

Might be good to build the expectation for users that they should look to the top of the module for most of the comments. That's where most of the info is likely to be I suspect, so that way users can get all the info rather than bits and pieces. Also it prevents us from having to maintain all these comments if they go out of date.

Reply:
I think that's a good strategy in general! In this particular case, there are enough concerns that I want to go overkill on the warnings. I'd really like to avoid having to take this toy away in the future 😅